fundamentals.js

```diff
@@ -0,0 +1,65 @@
+const fundamentalsRepoUrl = 'https://github.com/SAP/fundamental.git';
+
+const cmd = require('node-cmd');
+
+exports.importFundamentals = () => {
+
+    cmd.get(
+        `
+            git clone ${fundamentalsRepoUrl}
+        `,
+        (err, data, stderr) => {
+            if (!err) {
+                console.log('the Fiori Fundamentals repository cloned:\n\n', data)
+                cmd.get(
+                    `
+                        cd fundamental
+                        npm i
+                    `,
+                    (err, data, stderr) => {
+                        if (!err) {
+                            console.log('Finished installing npm packages:\n\n', data)
+                            cmd.get(
+                                `
+                                    cd fundamental/docs/
+                                    bundle exec jekyll build --config _config.yml,_mvx.yml --baseurl /_site
+                                `,
+                                (err, data, stderr) => {
+                                    if (!err) {
+                                        console.log('Finished building the static files:\n\n', data)
+                                    } else {
+                                        console.log('error', err)
+                                    }
+
+                                }
+                            );
+                        } else {
+                            console.log('error', err)
+                        }
+
+                    }
+                );
+            } else {
+                console.log('error', err)
+            }
+
+        }
+    );
+}
+
+exports.copyStaticFundamentals = () => {
+    cmd.get(
+        `
+            mkdir -p public/_site
+            cp -R fundamental/docs/_site/ public/_site
```

Comment:
We need to look into a better place to store these later.

Reply:
@ckoutsiaris - Can you import the fundamentals to `./static/docs/fundamental/`
Then gatsby build will copy them over to `public` folder
@MerlindlH Will look at adding selector on those in the CMS Admin

Reply:
@devinea  Done! I will create a new PR with only the static Fundamental files because there are 200+ and I want to keep this one light.

<img width="1180" alt="screenshot 2019-02-04 at 15 24 00" src="https://user-images.githubusercontent.com/44162302/52217724-f3ab4400-2890-11e9-97a6-46309b28f643.png">